orcid-web/src/main/webapp/static/javascript/readme.md

```diff
@@ -0,0 +1,45 @@
+# Refactoring JS libraries
```

Comment:
1. Should be called`README.md`.  
2. **DO**`**DO**` use markdown bold. 
3. Needs link to this from DEVSETUP.md.